fares-data-build-tool/fdbt-aws
# Feature/cfd 117 set up athena

# Description

-   Adds cloudformation template for Athena and updates S3 template to include Athena query results bucket

# Testing instructions

-   Check that you are able to run queries against the matching_data table while using the fdbt-athena-workgroup

# Type of change


# Checklist:
